src/player.h

```diff
@@ -1189,6 +1189,9 @@ class player : public Character, public JsonSerializer, public JsonDeserializer
 
         // crafting.cpp
         float lighting_craft_speed_multiplier( const recipe & rec ) const;
+        int time_to_craft( const recipe &rec, int batch_size = 1 );
+        std::vector<const item *> get_eligible_containers_for_crafting() const;
+        bool check_eligible_containers_for_crafting( const recipe &rec, int batch_size = 1 ) const;
```

Comment:
Agreed. It's a temporary place. I'm going to factor them [out](https://github.com/CleverRaven/Cataclysm-DDA/issues/20220#issuecomment-280862911) (not in this PR).